safe_transaction_service/account_abstraction/serializers.py

```diff
@@ -363,6 +352,13 @@ def validate(self, attrs):
                 f"SafeOperation with hash={safe_operation_hash_hex} does not exist"
             )
 
+        # Parse valid owners from init code
+        if user_operation_model.init_code:
+            decoded_init_code = decode_init_code(
+                bytes(user_operation_model.init_code), self.ethereum_client
+            )
+            self._deployment_owners = decoded_init_code.owners
```

Comment:
Thinking in the future what do you think about to have something like a `dataclass` for the `init_code` and refactor this, I could see that we are doing the same here [safe_transaction_service/account_abstraction/serializers.py:146](https://github.com/safe-global/safe-transaction-service/blob/229d4bf6a533e1819cb84ccf2c5d26c7a67c5f20/safe_transaction_service/account_abstraction/serializers.py#L146)

Reply:
What do you mean? `Decode` already returns a `dataclass`

Reply:
You are right, then makes sense to have this field `_deployment_owners` int he `SafeOperationSignatureValidatorMixin`? 
For me doesn't look part of this class, I would say that would be better use directly from the decode init code Dataclass.

Reply:
It has to be there, as the default `_get_owners` function is using it, so it needs to be declared